Source/Core/DolphinQt/Debugger/MemoryWidget.cpp

```diff
@@ -3,12 +3,14 @@
 
 #include "DolphinQt/Debugger/MemoryWidget.h"
 
+#include <fmt/printf.h>
```

Comment:
This include should be further down in its own group, between the STL headers and the Qt headers.